test/acceptance/utils/StubForWireMock.scala

```diff
@@ -46,6 +88,34 @@ trait StubForWireMock {
     )
   }
 
+  def verificationRequestJson(): String = {
+    """{
+      |"email":"b@a.com",
+      |"templateId" : "verifyEmailAddress",
+      |"templateParameters":{},
+      |"linkExpiryDuration":"P3D",
+      |"continueUrl":"/customs-email-frontend/email-verified"
+      |}
+    """.stripMargin
+  }
+
+  def stubVerificationRequest(url: String, response: String,status: Int): Unit = {
```

Comment:
do you really need to pass `response` as an argument to this function, as there is only one call to this function yet and empty string is being passed?